sky/backends/cloud_vm_ray_backend.py

```diff
@@ -4045,6 +4053,11 @@ def set_autostop(self,
         # The core.autostop() function should have already checked that the
         # cloud and resources support requested autostop.
         if idle_minutes_to_autostop is not None:
+            # Skip auto-stop for Kubernetes clusters.
+            if isinstance(handle.launched_resources.cloud, clouds.Kubernetes):
+                logger.info('Auto-stop is not supported for Kubernetes '
```

Comment:
Should we `assert controller_utils.Controllers.from_name(handle.cluster_name) == controller_utils.Controllers.JOBS_CONTROLLER` to ensure we do not accidentally get into this code path.